coef_forward: 1
coef_front_back: 1
coef_turn: 1
coef_up: 1
horizontal_left:
  pin: 9
  reversed: true
horizontal_right:
  pin: 6
  reversed: false
pitch_resist_coef: 50
ros_ip_address: 192.168.88.1
up_down_power_coef: 1
vertical_back:
  esc: 10
  reversed: true
vertical_left:
  pin: 5
  reversed: 'true'
vertical_right:
  pin: 3
  reversed: true
